lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -124,11 +112,26 @@ public function getExtension()
      */
     protected function getFilename($id)
     {
+        if ('\\' === DIRECTORY_SEPARATOR) {
```

Comment:
@Tobion just `bin2hex` every char in the key, IMO, and have that for both linux and windows. Sounds inefficient (probably is), but we can't try to discover case sensitivity anyway.